src/tests/fixtures.zig

```diff
@@ -0,0 +1,36 @@
+const std = @import("std");
+
+const Logger = @import("../server/logger.zig");
+const Config = @import("../server/config.zig");
+
+const TracingAllocator = @import("../server/tracing.zig");
+const PersistanceHandler = @import("../server/persistance.zig").PersistanceHandler;
+const MemoryStorage = @import("../server/storage.zig");
+
+const Fixtures = @This();
+
+allocator: std.mem.Allocator,
+tracing_allocator: TracingAllocator,
+logger: Logger,
+config: Config,
+persistance: PersistanceHandler,
+
+pub fn init() !Fixtures {
+    const allocator: std.mem.Allocator = std.testing.allocator;
+    const tracing_allocator: TracingAllocator = TracingAllocator.init(allocator);
+
+    const logger: Logger = try Logger.init(allocator, null, false);
+    const config: Config = try Config.load(allocator, null, null);
+    const persistance: PersistanceHandler = try PersistanceHandler.init(allocator, config, logger, null);
+
+    return Fixtures{ .allocator = allocator, .tracing_allocator = tracing_allocator, .logger = logger, .config = config, .persistance = persistance };
```

Comment:
Please, format this line by adding `,` after the last field.

Reply:
Done in [600ccd4.](https://github.com/sectasy0/zcached/pull/8/commits/600ccd45dce6b70fa905d5e7e18855827477dca5)